packages/react/src/analytics/integrations/GTM/utils.js

```diff
@@ -54,13 +55,24 @@ export const getContextParameters = context => ({
  * @returns {Array} - The product categories.
  */
 export const getProductCategory = categories => {
-  if (isArray(categories)) {
-    return categories;
-  }
+  let productCategories = isArray(categories)
+    ? [...categories]
+    : isString(categories)
+    ? categories.split('/')
+    : [];
```

Comment:
What do you think about spreading it in more lines and variables to improve the readability?

Because nested ternary makes the code more complex to read. 
Does it make sense to you?

Reply:
Makes sense! Fixed 👍🏼